operations/CSM_product_management/Post_Install_Customizations.md

```diff
@@ -516,41 +596,47 @@ A similar flow can be used to update the volume size for `cray-sls-postgres`, `g
 
    Example output:
 
-   ```
+   ```text
    NAME                TEAM       VERSION   PODS   VOLUME   CPU-REQUEST   MEMORY-REQUEST   AGE   STATUS
    cray-smd-postgres   cray-smd   11        3      100Gi     500m          8Gi              45m  Running
    ```
 
-1. If the status on the above command is `SyncFailed` instead of `Running`, refer to *Case 1* in the *SyncFailed* section of [Troubleshoot Postgres Database](../kubernetes/Troubleshoot_Postgres_Database.md#syncfailed). At this point the Postgres cluster is healthy, but additional steps are required to complete the resize of the Postgres PVCs.
+1. If the status on the above command is `SyncFailed` instead of `Running`, refer to *Case 1* in the
+   `SyncFailed` section of [Troubleshoot Postgres Database](../kubernetes/Troubleshoot_Postgres_Database.md#syncfailed).
 
-1. **This step is critical.** Store the modified `customizations.yaml` in the `site-init` repository in the customer-managed location. If not done, these changes will not persist in future installs or upgrades.
+   At this point the Postgres cluster is healthy, but additional steps are required to complete the resize of the Postgres PVCs.
+
+1. **This step is critical.** Store the modified `customizations.yaml` in the `site-init` repository in the customer-managed location.
+
+   If this is not done, these changes will not persist in future installs or upgrades.
 
    ```bash
    ncn# kubectl delete secret -n loftsman site-init
```

Comment:
Could you also pretty please remove these prompts to make this more copy-paste friendly? https://github.com/Cray-HPE/docs-csm/blob/MTL-1695/introduction/documentation_conventions.md#step-examples